### PR TITLE
doc: fix typo in multidisk document

### PR DESCRIPTION
**Pull request by CNLHC**
related to https://github.com/longhorn/website/issues/701

### CODE REVIEW

**Review by ejweber — CHANGES_REQUESTED** [review]
Thanks for the contribution, @CNLHC!

It looks like we documented the new default of 200 (changed from 500) on the Settings Reference page [as early as](https://github.com/longhorn/website/blob/d1e0f0e1766d7f332d729ebe33d3e61f3197f2fd/content/docs/archives/1.0.0/references/settings.md?plain=1#L154) `v1.0.0`, but we missed modifying this file. Would you be willing to make this change to `content/docs/<every_version_since_and_including_v1.0.0` so users on older versions aren't confused?

**Comment by CNLHC** [comment]
@ejweber sure. I have changed all the typos since `v1.0.0`

File: content/docs/1.5.0/volumes-and-nodes/multidisk.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Once those two conditions are met, you should be allowed to remove the disk.
 ## Configuration
 There are two global settings affect the scheduling of the volume.
 
-- `StorageOverProvisioningPercentage` defines the upper bound of `ScheduledStorage / (MaximumStorage - ReservedStorage)`. The default value is `500` (%). That means we can schedule a total of 750 GiB Longhorn volumes on a 200 GiB disk with 50G reserved for the root file system. Because normally people won't use that large amount of data in the volume, and we store the volumes as sparse files.
+- `StorageOverProvisioningPercentage` defines the upper bound of `ScheduledStorage / (MaximumStorage - ReservedStorage)`. The default value is `200` (%). That means we can schedule a total of 750 GiB Longhorn volumes on a 200 GiB disk with 50G reserved for the root file system. Because normally people won't use that large amount of data in the volume, and we store the volumes as sparse files.
```
Comment (innobead) [inline content/docs/1.5.0/volumes-and-nodes/multidisk.md:46]: From 1.5.0, we changed the provision percentage to 100 instead.

**Review by innobead — CHANGES_REQUESTED** [review]
Just one change required, others LGTM.

**Review by innobead — APPROVED** [review]

**Pull request merged**